test/functional/logos_rpc_getrawunsolvedblock.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Logos Foundation
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Tests for Lotus `getrawunsolvedblock` rpc call
+"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_array_result,
+)
+from test_framework.messages import (
+    CBlock,
+    FromHex,
+)
+
+from decimal import Decimal
+
+import pprint
+
+NUM_TRANSACTIONS = 10
+
+pp = pprint.PrettyPrinter(depth=6)
+
+
+class LotusGetRawUnsolvedBlock(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
```

Comment:
Just remove this line.

Also, isn't it simpler with a setup chain? I think you can do "sendtoaddress" right away IIRC.

Reply:
You can't mine without a second node.

Reply:
Just checked, using the cached chain is actually slower by a couple seconds in the test. I had based this off mining basic originally.

Reply:
> You can't mine without a second node.

Really? It always worked for me, it's working on my regtest with just 1 node